src/dialects/mssql/index.js

```diff
@@ -25,6 +25,12 @@ export class MssqlDialect extends AbstractDialect {
       defaultValue: false,
       update: false,
     },
+    addColumn: {
```

Comment:
It seems that mssql does actually support this; https://learn.microsoft.com/en-us/sql/t-sql/statements/alter-table-transact-sql?view=sql-server-2017

Reply:
same for this as mariadb?